layer/nrlf/core/tests/test_validators.py

```diff
@@ -722,6 +722,40 @@ def test_validate_ssp_content_without_any_context_related():
 
 
 def test_validate_ssp_content_without_asid_in_context_related():
+    validator = DocumentReferenceValidator()
+    document_ref_data = load_document_reference_json("Y05868-736253002-Valid")
+
+    document_ref_data["context"]["related"] = [
+        {
+            "identifier": {
+                "system": "https://fhir.nhs.uk/Id/nhsSpineASID",
+                "value": "1234",
+            }
+        }
+    ]
+
+    result = validator.validate(document_ref_data)
+
+    assert result.is_valid is False
+    assert len(result.issues) == 1
+    assert result.issues[0].dict(exclude_none=True) == {
+        "severity": "error",
+        "code": "value",
+        "details": {
+            "coding": [
+                {
+                    "system": "https://fhir.nhs.uk/ValueSet/Spine-ErrorOrWarningCode-1",
+                    "code": "INVALID_IDENTIFIER_VALUE",
+                    "display": "Invalid identifier value",
+                }
+            ]
+        },
+        "diagnostics": "Invalid ASID value '1234'. context.related must contain a single valid ASID identifier when content contains an SSP URL",
+        "expression": ["context.related[0].identifier.value"],
+    }
+
+
+def test_validate_asid_with_no_ssp_content():
```

Comment:
I don't think this test name represents the test, It's called `test_validate_asid_with_no_ssp_content` but the docref in the test does have SSP content but has no ASID.